meteor/client/styles/rundownView.scss

```diff
@@ -260,6 +260,10 @@ body.no-overflow {
 				text-align: left;
 			}
 
+			&.show-style {
+				width: 40%
```

Comment:
Why specifically `40%`? I think this would work much better if you used CSS grid, with ellipsis on the bottom row. That way the columns would always show full headers, but the column widths would be distributed based on the length of the names, which is important if Variant name would be much longer than Base name (or vice versa).

Reply:
I found that the spans containing the `Show Style` and `Show Style Variant`, had to have a width of `44%` or lower, otherwise they would split into multiple lines, if name is too long, even though i have applied the overflow ellipsis.

`40%` just seemed like a clean number, as it is divisible by 5.

I have not applied the overflow ellipsis to the `Show Style` and `Show Style Variant` headers, so they should always show in full, regardless of the length of the variant or base.

Does that answer your concerns?

Reply:
It's probably good enough for our needs, but it seems like that 44% depends on the margin that is set on the `timing-clock`, which is `1` or `1.2em`. So it feels like on very small or very large screens this will not look ideal and might break

Reply:
There are so many things that seem to break on small/large screens, so this one would not be the only one that would need changes, if thats your concern. Anyways thanks for your feedback.